### ISSUE TITLE
Almost constant train evaluation

### ISSUE THREAD

**Issue by dandrane** [DataArk/CBLUE-Baseline#1]
When running `CHIP-CDN-PredictTextsim.ipynb model. fit()`, the variation in loss is minimal and the train evaluation is almost constant. By increasing the learning rate to 0.1, we can see the change in loss, but the train evaluation still fluctuates steadily above 0.50.
<img width="328" alt="image" src="https://user-images.githubusercontent.com/32335215/228734215-afcade04-d5b5-454c-bcac-d602a34b9bea.png">

When changing to loss function to `focal loss`, still the same.
I want to know if anything is wrong with this and how to adjust it, thank you!
